Tree_Preorder_Traversal/Tree_Preorder_Traversal.js.js

```diff
@@ -0,0 +1,93 @@
+//Tree_Preorder_Traversal in javascript
+class Node
+{
+  constructor(data)
```

Comment:
Please add 4 space indentation at the start of each block.

Reply:
4 space indentation Added